street-comics-api/app/api/comics_api/v1/adapters/marvel/models/characters_params.rb

```diff
@@ -9,18 +9,21 @@ class CharactersParams
             attr_reader(:limit)
             attr_reader(:offset)
             attr_reader(:order_by)
+            attr_reader(:name_starts_with)
 
             def initialize(params)
               @limit = params.fetch(:limit)
               @offset = params.fetch(:offset)
               @order_by = params.fetch(:order_by)
+              @name_starts_with = params.fetch(:name_starts_with)
             end
 
             def to_hash
               {
                 limit: limit,
                 offset: offset,
-                orderBy: order_by
+                orderBy: order_by,
+                nameStartsWith: name_starts_with,
```

Comment:
Started debating with myself if this mapping/conversion should be handled inside the model.

It seems better to handle that in the adapter layer.

At the same time, this model is marvel-specific, so it's probably not going to have undesirable impacts for now.

Will keep a thread in the background to think about that in future refactorings.